src/renderer/utils/readableStream.ts

```diff
@@ -0,0 +1,87 @@
+import { Readable } from "readable-stream";
```

Comment:
```suggestion
import { Readable } from "stream";
```

> This package is a mirror of the streams implementations in Node.js.
>
> If you want to guarantee a stable streams base, regardless of what version of Node you, or the users of your libraries are using, use readable-stream only and avoid the "stream" module in Node-core, for background see this blogpost.

Since we bundle app with Node.js `v12.x` maybe we could use native `stream` module?
https://nodejs.org/dist/latest-v12.x/docs/api/stream.html#stream_readable_streams

Reply:
Makes sense, I'll do that in a separate PR.